modules/stochastic_tools/include/tests/DistributionValuePostprocessor.h

```diff
@@ -0,0 +1,46 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#ifndef DISTRIBUTIONVALUEPOSTPROCESSOR_H
+#define DISTRIBUTIONVALUEPOSTPROCESSOR_H
+
+#include "GeneralPostprocessor.h"
+
+class DistributionValuePostprocessor;
+class Distribution;
+
+template<>
+InputParameters validParams<DistributionValuePostprocessor>();
+
+/**
+ * This postprocessor is used to test the distribution capabilities.
+ */
+class DistributionValuePostprocessor : public GeneralPostprocessor
```

Comment:
Can you also change the name or this class to `TestDistributionPostprocessor` so when it is shown in a list that it is obvious that it is for testing.